### PR TITLE
Deploy production version of app-project to kubernetes

### PR DESCRIPTION
**Pull request by rogerhutchings**
- Deploy production version of app-project to kubernetes
- Rename the staging version for clarity

### CODE REVIEW

File: kubernetes/deployment.tmpl [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
       containers:
         - name: fe-project-staging
           image: zooniverse/fe-project:__IMAGE_TAG__
           ports:
             - containerPort: 3000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: front-end-project-production
```
Comment (camallen) [inline kubernetes/deployment.tmpl]: `front-end-project-production` doesn't really tell me which service this, could this be better named, e.g. `zooniverse-org-production`

Comment (adammcmaster) [reply kubernetes/deployment.tmpl]: I agree. At the moment it's obvious that the "front end project" is the rebuild, but that won't make sense a couple of years from now.

**Comment by adammcmaster** [comment]
> Will changing a service name require us to manually remove the service and re-create it?

Yes I think so – we won't have to manually create the new one but we'll need to remove the old one.

File: kubernetes/deployment.tmpl (hunk 1 above)
Comment (rogerhutchings) [reply kubernetes/deployment.tmpl]: Okay - how does that fit in with specifying that this is the project app as opposed to the static pages / admin / whatever app, as these will be built out separately? Or will they share the same service?

**Comment by rogerhutchings** [comment]
> With this setup, production and staging will both always be deployed at the same time and will always run the same version of the code. Is that what we want at this stage?

I _think_ so - what we'd want ideally is automatic branch builds at some point for testing purposes, but I'll settle for a staging and production build

File: kubernetes/deployment.tmpl (hunk 1 above)
Comment (camallen) [reply kubernetes/deployment.tmpl]: `zooniverse-org-project-production` vs `zooniverse-org-static-production` etc?

Comment (rogerhutchings) [reply kubernetes/deployment.tmpl]: Yeah - `zooniverse.org` will consist of multiple apps, which will have >1 environment (staging, production) etc, so would `zooniverse-org-project-production` for this one be more appropriate?

**Comment by rogerhutchings** [comment]
I'm game - let me see if I can get `kubectl` set up

**Pull request merged**